src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -99,16 +95,22 @@ public void onCommand(Member sender, CommandContext context) {
                 builder.addField(context.i18n("commands.userinfo.activity"), gameStatus, true);
             }
 
-            Table.TableBuilder tableBuilder = new Table.TableBuilder(context.i18n("commands.userinfo.role_id"), context.i18n("commands.userinfo.role_name"));
-
+            StringBuilder stringer = new StringBuilder();
+            String author = "Roles for: " + sender.getUser().getAsTag();
+            EmbedBuilder embed = new EmbedBuilder(MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, sender.getUser()).setAuthor(author));
+            int i = 1;
             for (Role role : member.getRoles()) {
-                tableBuilder.addRow(role.getId(), role.getName());
+                if (i % 20 == 0) {
+                    pageList.add(new PageObjects.EmbedPage(
+                            new EmbedBuilder(MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, sender.getUser()).setAuthor(author).setAuthor(author)),
+                            false));
+                    stringer = new StringBuilder();
+                }
+                stringer.append("<@&" + role.getId() + ">" + " ID: " + role.getId() + "\n");
```

Comment:
no. just no. we don't want to mention roles every time someone checks userinfo

Reply:
Pings don't work in embeds...?

Reply:
That could easily change in the future, and pining in this doesn't make seance when a name would do just fine.

Reply:
I disagree. Mentions have never and probably will never work in embeds so I think this is a nice way to do it. It means that we have colour with it as well. Humans work on colour 😄

Reply:
I still think they could change it, and also well yes it does show the color that's all it does extra, I'd prefer to give a color value then to show the color. also what was wrong with using the table?

Reply:
The tables are broken on mobile; this isn't. I also feel like adding colour is a good thing, and we shouldn't take it away because "that's all it does".

Reply:
Tables looked fine on mobile for me. in fact embeds are broken on mobile for me atm. I'm still under the opinion mentioning is a bad idea.

Reply:
Also to get to that page, we do edit the message which is something else that does not ping
Anyways I'll let you 2 come to a conclusion about this

Reply:
It pining isn't my only point. a name does just fine. doesn't have the @ symbol in front of it. discord could change it where editing, or embeds will ping in the future. plus like i mentioned embeds are broken on android atm so the table even works better on mobile atm.

Reply:
> plus like i mentioned embeds are broken on android atm so the table even works better on mobile atm.

Even so, this should not be a reason as it's only broken temporarily. Tables have always been broken on mobile due to the screen width not being wide enough on some models. Because of this, we should really avoid tables when we can.

>  a name does just fine. doesn't have the @ symbol in front of it. 
The @ symbol is a bit distracting yeah but I still think it looks better than a plain name. Simply because it has the colour that allows you the mentally link the role in discord with the role we are displaying. Especially on guilds where people might not know all the role names, a colour helps link it better.

> discord could change it where editing
Discord could do anything, it's not really our responsibility to try and predict it. 

I'll also add that we are on of the only bots that use tables and the reason for that is because they look so terrible if the screen size isn't big enough (Not so much of an issue here but in larger tables it can be) so I still definitely think that we should avoid tables and find better ways to represent data.

Reply:
I think it probably depends on more than just how it looks on android alone, because embeds look fine & even tables are almost how they should be on my phone, it'll probably depend on other factors like screen size
/shrug

Reply:
Tables work completely fine if they contain smaller sets of data, and are a lot more compact displaying wise. With this change user info will store a lot less info per page. We have a table system why are we avoiding using it for things it's good at like this?
I do agree it would be nice to display the color (along with it's hex code) but that doesn't mean we should compromise other stuff just for that.

Reply:
Tables are actually less compact using the method in the PR because we don't have any of the headers or anything like that. Also, it's good at it yeah but, as previously mentioned, we have had reports of users on smaller screen sizes not being able to read it properly. I'm not saying we should avoid the table system for things where we can't do anything else but we **can** display it a different way here. 

What do you think we're compromising with this method? We're displaying the exact same information with the colour as an added bonus.

Also, I don't think we should display a hex code here as it's used to see what roles a user has, not what colour a role is. The `_roles` command (Which we do use a table for and I think that's the best way) is for that.

Reply:
>Tables are actually less compact using the method in the PR because we don't have any of the headers or anything like that.
not in terms of the amount on a page, not to mention this way things don't line up as nicely

> we have had reports of users on smaller screen sizes not being able to read it properly
for this specific command or just commands in general.
on this specific command cause that makes a huge difference

>What do you think we're compromising with this method? We're displaying the exact same information with the colour as an added bonus.
1. looks less organized as the ids arn't even spaced from the role name
2. the @ symbol is ugly and could be confused for being part of the role name

Reply:
For this specific command, from Jip.

I'll give you the alignment and confusion, those are valid points.